src/pdfkit.cs

```diff
@@ -509,21 +1321,49 @@ interface PdfDocument : NSCopying {
 
 		[Export ("selectionFromPage:atCharacterIndex:toPage:atCharacterIndex:")]
 		PdfSelection GetSelection (PdfPage startPage, nint startCharIndex, PdfPage endPage, nint endCharIndex);
+
+		[NoiOS]
+		[Mac (10,7)]
+		[Export ("printOperationForPrintInfo:scalingMode:autoRotate:")]
+		[return: NullAllowed]
+		NSPrintOperation GetPrintOperation ([NullAllowed] NSPrintInfo printInfo, PdfPrintScalingMode scaleMode, bool doRotate);
 	}
-	
-	[BaseType (typeof (NSObject))]
+
+	[iOS (11,0)]
+	[BaseType (typeof (NSObject), Name = "PDFDocumentDelegate")]
+#if IOS
+	[Protocol]
```

Comment:
@rolfbjarne  mmm let me doublecheck the generated code because then we have some wrong bindings, shouldn't it get it from BaseType if it exists?